src/styles/blocks/_card.scss

```diff
@@ -0,0 +1,70 @@
+.card {
+  box-sizing: border-box;
```

Comment:
move it to the * selector

Reply:
fix it
![image](https://user-images.githubusercontent.com/98764641/211784945-b149ad10-c1fe-4693-9764-0f798510d114.png)